src/cert/ssl_cert_provider.cpp

```diff
@@ -109,8 +109,19 @@ long random_long()
 {
     long out{0};
     std::array<uint8_t, 4> bytes;
+
+#ifdef MULTIPASS_COMPILER_GCC
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+#endif
+
+    // The array must be used uninitialized. Otherwise, there will be no randomness.
```

Comment:
I'm afraid this does not sound right. `RAND_bytes` is what generates the randomness. Relying on whatever was on memory would be undefined behavior, a poor randomness source, and a security hazard. The array is left uninitialized because it is written to before it is ever read from. The goal is just performance (avoid writing zeros just before overwriting them). 

Could you please remove?

Reply:
Oops, right, the comment was entirely wrong. Removed.